stdlib/public/Concurrency/AsyncDropWhileSequence.swift

```diff
@@ -14,6 +14,32 @@ import Swift
 
 @available(SwiftStdlib 5.5, *)
 extension AsyncSequence {
+  /// Omits elements from the base asynchronous sequence until a given closure
+  /// returns false, after which it passes through all remaining elements.
+  ///
+  /// Use `drop(while:)` to omit elements from an asynchronous sequence until
+  /// the element received meets a condition you specify.
+  ///
+  /// In this example, an asynchronous sequence called `Counter` produces `Int`
+  /// values from `1` to `10`. The `drop(while:)` function causes the modified
+  /// sequence to ignore the values `1` through `4`, and instead emit
+  /// `5` through `10`:
+  ///
+  ///     for try await number in Counter(howHigh: 10)
+  ///             .drop(while: { $0 < 5 } ) {
+  ///         print("\(number) ", terminator: " ")
+  ///     }
+  ///     // prints "5 6 7 8 9 10"
+  ///
+  /// After the predicate returns `false`, the sequence never executes it again,
+  /// and from then on the sequence passes through elements from its underlying
+  /// sequence as-is.
```

Comment:
perhaps a predicate that illustrates this might be good; e.g. `$0 % 3 != 0` or something along those lines - that way folks can understand that it is a flipped trigger

Reply:
Yep, changed to that. Thanks.